src/components/dialog-full-screen/dialog-full-screen-test.stories.tsx

```diff
@@ -141,6 +143,39 @@ Nested.parameters = {
   },
 };
 
+export const WithFullWidthHeader: StoryType = {
+  render: (args) => {
+    const { children, ...rest } = args;
+    return <DialogFullScreen {...rest}>{children}</DialogFullScreen>;
+  },
+  args: {
+    children: "Content",
+    open: true,
+    title: (
+      <Box maxWidth="750px" width="100%" data-testid="test">
+        <StepFlow
+          category="category"
+          title="title"
+          currentStep={1}
+          totalSteps={6}
+          showProgressIndicator
+        />
+      </Box>
+    ),
+    showCloseIcon: false,
+    onCancel: () => {},
+    hasFullWidthHeader: true,
+  },
+  decorators: [
+    (Story) => (
+      <div style={{ height: 900, width: "100%" }}>
```

Comment:
**nitpick/question:** Such a minor thing but could we use our `Box` component here instead of a `div`?

Reply:
Replaced the `div` with `Box`.